markdowns/1.md

```diff
@@ -7,39 +7,28 @@ date: "2020-09-03"
 id: "tutorial1md"
 --->
 
-# Build Your First QA System
+# Build Your First Question Answering System
 
-<img style="float: right;" src="https://upload.wikimedia.org/wikipedia/en/d/d8/Game_of_Thrones_title_card.jpg">
+- **Level**: Beginner
+- **Time to complete**: 20 minutes
+- **Prerequisites**: Prepare the Colab environment. See links below.
+- **Nodes Used**: `ElasticsearchDocumentStore`, `BM25Retriever`
+- **Goal**: After completing this tutorial, you will have built a question answering pipeline that can answer questions about the Game of Thrones series.
 
-[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/deepset-ai/haystack-tutorials/blob/main/tutorials/01_Basic_QA_Pipeline.ipynb)
+This tutorial teaches you how to set up a question answering system that can search through complex knowledge bases, such as an internal wiki or a collection of financial reports. We will work on a set of Wikipedia pages about Game of Thrones. Let's learn how to build a question answering system and discover more about the marvellous seven kingdoms!
 
-Question Answering can be used in a variety of use cases. A very common one:  Using it to navigate through complex knowledge bases or long documents ("search setting").
 
-A "knowledge base" could for example be your website, an internal wiki or a collection of financial reports. 
-In this tutorial we will work on a slightly different domain: "Game of Thrones". 
 
-Let's see how we can use a bunch of Wikipedia articles to answer a variety of questions about the 
-marvellous seven kingdoms.
+## Preparing the Colab Environment
 
+- [Enable GPU Runtime in GPU](https://docs.haystack.deepset.ai/v5.2-unstable/docs/enable-gpu-runtime-in-colab)
```

Comment:
This would be nice. I see that the details tag works in colab. But It doesn't work in jupyter notebook which makes me a bit hesitant to go ahead with this change.